src/Build/Evaluation/Expander.cs

```diff
@@ -3506,7 +3506,7 @@ private static void ConstructFunction(IElementLocation elementLocation, string e
                             functionArguments = ExtractFunctionArguments(elementLocation, expressionFunction, argumentsContent);
                         }
 
-                        remainder = expressionFunction.Substring(argumentsEndIndex + 1);
+                        remainder = expressionFunction.Substring(argumentsEndIndex + 1).Trim();
```

Comment:
I'm a bit concerned about allocations here but this should only cost a new `string` in the case where there _is_ whitespace so that should be ok.